kahuna/public/js/directives/gr-remember-scroll-top.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+import Rx from 'rx';
+import 'rx-dom';
+
+import {rxUtil} from '../util/rx';
+
+export const rememberScrollTop = angular.module('gr.rememberScrollTop', [
+    rxUtil.name
+]);
+
+rememberScrollTop.directive('grRememberScrollTop', ['subscribe$', function (subscribe$) {
+    const storageKey = 'gr:rememberScrollTop';
+
+    return {
+        restrict: 'A',
+        link: function(scope, element, attrs) {
+            // Only remember if attribute if truthy. This is an annoying workaround to the
+            // inability to add directives conditionally
+            if (scope.$eval(attrs.grRememberScrollTop)) {
```

Comment:
An [HTML5-esque](https://github.com/guardian/grid/blob/master/kahuna/public/js/components/gr-icon/gr-icon.js#L20) way of doing this by testing the presence of the attribute.